### PR TITLE
Improve Generic x86-64 installation instructions

### LINKED ISSUE #17848: Installation instructions for NUC do not work
### Feedback

I followed the NUC installation instructions, creating the USB stick and attempting to boot. This fails reproducibly (see #1358). It appears the documentation does not cover the actual NUC install. I was able to make it work by starting a Linux live distribution.

First, create a USB stick with your favourite Linux live distro (I happened to have an Ubuntu live stick) and boot from that.

Download hassos_intel-nuc-5.13.img.xz (you can do this from the Linux live distro; I happened to have that file on another USB stick so I inserted that and copied it to /tmp/).

    sudo -i
    lsblk                     # this shows me that the mSATA disk is at /dev/sda
    cp /media/ubuntu/Sandisk16G/has* /tmp/   # Copy and unpack the hassos img.xz
    cd /tmp
    xz -d hassos_intel-nuc-5.13.img.xz
    dd if=/tmp/hassos_intel-nuc-5.13.img of=/dev/sda bs=4M conv=fsync

Now power down the Linux live distribution, remove the USB stick(s) and switch on the NUC again. You should now see Home Assistant booting. Access http://homeassistant.local:8123/ to continue the onboarding.

Done!

### URL

https://www.home-assistant.io/installation/nuc/

### Version

2021.5.4

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by agners**
## Proposed change

Mainly add information about BIOS and how a Live system can be used to flash the OS image.

It also converts the Generic x86-64 to be a "board" type instruction. This allows to reuse the Etcher workflow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: closes #17848

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: _config.yml
Comment (ludeeus) [inline _config.yml]:
```suggestion
      installation_media: "storage device"
```
Is there any other possible device? 😄 
How about `"storage device"` as that covers all current and future posibilities?

File: source/_includes/installation/operating_system.md
Comment (ludeeus) [inline source/_includes/installation/operating_system.md]:
```suggestion
This guide assumes that you have a dedicated generic x86 PC (typically an Intel or AMD-based system) available to exclusively run Home Assistant Operating System. The system must be 64-bit capable and able to boot using UEFI. Pretty much all systems produced in the last 10 years support the UEFI boot mode.
```

File: source/_includes/installation/operating_system.md
Comment (ludeeus) [inline source/_includes/installation/operating_system.md]:
```suggestion
As a next step, we need to write the Operating System image to the installation media. Depending on your system this can be a S-ATA hard disk, S-ATA SSD, a M.2 SSD or even a eMMC. Home Assistant Operating System has no integrated installer. You need to use your Desktop computer (e.g. by using a USB to S-ATA adapter) or boot a live operating system on your target system to install Home Assistant Operating System.
```

File: source/_includes/installation/operating_system.md
Comment (ludeeus) [inline source/_includes/installation/operating_system.md]:
```suggestion
2. Make sure an ethernet cable for network is plugged in
3. Power the system on. 
```

**Review by ludeeus — DISMISSED** [review]
Content-wise I think this is a good addition 👍 
As for the wording of it, I would like a second pair of eyes on it before it's merged

File: source/_includes/installation/operating_system.md
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
To boot Home Assistant OS the BIOS needs to have UEFI boot mode enabled and Secure Boot disabled. The following screenshots are from a 7th generation Intel NUC system. The BIOS menu will likely look different on your systems. However, the options should still be present and named similarly.
```

File: source/_includes/installation/operating_system.md
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
If you prefer to use a live operating system, follow the instructions of your Live distribution (e.g., [this Ubuntu guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Once you booted the live operating system, the following steps on how-to write the image to your installation media can be followed.
```

File: source/_includes/installation/operating_system.md
Comment (frenck) [inline source/_includes/installation/operating_system.md]:
```suggestion
1. If you used your Desktop system to write to your installation media, install the installation media ({{site.installation.types[page.installation_type].installation_media}}) to the target system. Otherwise, shutdown the live operating system and make sure to remove the USB flash drive you have been using for the live system.
```

File: source/_includes/installation/operating_system.md
Comment (frenck) [inline source/_includes/installation/operating_system.md]: Very minor, but I think "board" is a bit tech :)

```suggestion
Follow this guide if you already are running a supported virtual machine hypervisor. If you are not familiar with virtual machines we recommend installation Home Assistant OS directly on a [Raspberry Pi](/installation/raspberrypi) or an [ODROID](/installation/odroid).
```

File: _config.yml
Comment (agners) [reply _config.yml]: Hehe, well, whatever PC BIOS support, the do support quite a bit. Storage device is fine for me.

**Review by frenck — APPROVED** [review]

**Pull request merged**
